2022-04-19 22:21:57,523 INFO    MainThread:12550 [wandb_setup.py:_flush():75] Loading settings from /home/ido/.config/wandb/settings
2022-04-19 22:21:57,523 INFO    MainThread:12550 [wandb_setup.py:_flush():75] Loading settings from /home/ido/projects/MNIST_comparison/wandb/settings
2022-04-19 22:21:57,523 INFO    MainThread:12550 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-04-19 22:21:57,523 INFO    MainThread:12550 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'train_denoiser.py', 'program': '/home/ido/projects/MNIST_comparison/train_denoiser.py'}
2022-04-19 22:21:57,523 INFO    MainThread:12550 [wandb_init.py:_log_setup():405] Logging user logs to /home/ido/projects/MNIST_comparison/wandb/run-20220419_222157-2pilu5nz/logs/debug.log
2022-04-19 22:21:57,523 INFO    MainThread:12550 [wandb_init.py:_log_setup():406] Logging internal logs to /home/ido/projects/MNIST_comparison/wandb/run-20220419_222157-2pilu5nz/logs/debug-internal.log
2022-04-19 22:21:57,523 INFO    MainThread:12550 [wandb_init.py:init():439] calling init triggers
2022-04-19 22:21:57,523 INFO    MainThread:12550 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {}
2022-04-19 22:21:57,523 INFO    MainThread:12550 [wandb_init.py:init():458] re-initializing run, found existing run on stack: 1q27mc2v
2022-04-19 22:21:57,523 INFO    MainThread:12550 [wandb_run.py:finish():1539] finishing run ibenshaul/MNIST_trial_experiments/1q27mc2v
2022-04-19 22:21:57,523 INFO    MainThread:12550 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-04-19 22:21:57,523 INFO    MainThread:12550 [wandb_run.py:_restore():1674] restore
2022-04-19 22:22:00,487 INFO    MainThread:12550 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 5269
  total_bytes: 5269
}

2022-04-19 22:22:00,597 INFO    MainThread:12550 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 5269
  total_bytes: 5269
}

2022-04-19 22:22:01,015 INFO    MainThread:12550 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 5269
  total_bytes: 5269
}

2022-04-19 22:22:01,217 INFO    MainThread:12550 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 5269
  total_bytes: 40347
}

2022-04-19 22:22:01,319 INFO    MainThread:12550 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 5269
  total_bytes: 40347
}

2022-04-19 22:22:01,421 INFO    MainThread:12550 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 5269
  total_bytes: 40347
}

2022-04-19 22:22:01,522 INFO    MainThread:12550 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 5269
  total_bytes: 40347
}

2022-04-19 22:22:01,624 INFO    MainThread:12550 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 5269
  total_bytes: 40347
}

2022-04-19 22:22:01,725 INFO    MainThread:12550 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 40347
  total_bytes: 40347
}

2022-04-19 22:22:01,827 INFO    MainThread:12550 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 40347
  total_bytes: 40347
}

2022-04-19 22:22:01,929 INFO    MainThread:12550 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 40347
  total_bytes: 40347
}

2022-04-19 22:22:02,031 INFO    MainThread:12550 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 40347
  total_bytes: 40347
}

2022-04-19 22:22:02,132 INFO    MainThread:12550 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 40347
  total_bytes: 40347
}

2022-04-19 22:22:02,234 INFO    MainThread:12550 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 40347
  total_bytes: 40347
}

2022-04-19 22:22:02,336 INFO    MainThread:12550 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 40347
  total_bytes: 40347
}

2022-04-19 22:22:02,879 INFO    MainThread:12550 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 40347
  total_bytes: 40347
}

2022-04-19 22:22:03,686 INFO    MainThread:12550 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 40347
  total_bytes: 40347
}
local_info {
}

2022-04-19 22:22:04,791 INFO    MainThread:12550 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-04-19 22:22:04,791 INFO    MainThread:12550 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-04-19 22:22:04,793 INFO    MainThread:12550 [wandb_run.py:_footer_sync_info():2822] logging synced files
2022-04-19 22:22:04,793 INFO    MainThread:12550 [wandb_init.py:init():492] starting backend
2022-04-19 22:22:04,793 INFO    MainThread:12550 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-04-19 22:22:04,796 INFO    MainThread:12550 [backend.py:ensure_launched():219] starting backend process...
2022-04-19 22:22:04,803 INFO    MainThread:12550 [backend.py:ensure_launched():224] started backend process with pid: 13832
2022-04-19 22:22:04,804 INFO    MainThread:12550 [wandb_init.py:init():501] backend started and connected
2022-04-19 22:22:04,807 INFO    MainThread:12550 [wandb_init.py:init():565] updated telemetry
2022-04-19 22:22:04,808 INFO    MainThread:12550 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-04-19 22:22:05,227 WARNING MainThread:12550 [wandb_init.py:init():975] interrupted
Traceback (most recent call last):
  File "/home/ido/miniconda3/envs/sparsity_probe_env/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 954, in init
    run = wi.init()
  File "/home/ido/miniconda3/envs/sparsity_probe_env/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 597, in init
    run_result = backend.interface.communicate_run(run, timeout=30)
  File "/home/ido/miniconda3/envs/sparsity_probe_env/lib/python3.9/site-packages/wandb/sdk/interface/interface.py", line 210, in communicate_run
    return self._communicate_run(run, timeout=timeout)
  File "/home/ido/miniconda3/envs/sparsity_probe_env/lib/python3.9/site-packages/wandb/sdk/interface/interface_shared.py", line 320, in _communicate_run
    resp = self._communicate(req, timeout=timeout)
  File "/home/ido/miniconda3/envs/sparsity_probe_env/lib/python3.9/site-packages/wandb/sdk/interface/interface_shared.py", line 222, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/home/ido/miniconda3/envs/sparsity_probe_env/lib/python3.9/site-packages/wandb/sdk/interface/router.py", line 37, in get
    is_set = self._object_ready.wait(timeout)
  File "/home/ido/miniconda3/envs/sparsity_probe_env/lib/python3.9/threading.py", line 574, in wait
    signaled = self._cond.wait(timeout)
  File "/home/ido/miniconda3/envs/sparsity_probe_env/lib/python3.9/threading.py", line 316, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
